jsonnet/telemeter/prometheus/kubernetes/kubernetes.libsonnet

```diff
@@ -20,6 +20,8 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       rules: { groups: [] },
       htpasswdAuth: '',
       sessionSecret: '',
+      sar: '{"resource": "namespaces", "verb": "get", "resourceName": "telemeter-production", "namespace": "telemeter-production"}',
```

Comment:
also, rather than having to cofigure both `$._config.namespace` as well as the SAR so that the namespaces match, the SAR should inherit the namespace to which the object is deployed.

Reply:
ah, that's indeed a good idea